### PR TITLE
Toast components

### PR DESCRIPTION
**Pull request by andreivdev**
### What does it do?

Adds 3 new components for toasts according to latest Figma designs

### Any helpful background information?

Added 3 buttons for easy visualisation of the toasts during development. These buttons should be taken out at release.

### Any new dependencies? Why were they added?

No

### Relevant screenshots/gifs

Screenshot 2022-01-27 at 18 10 21

### Does it close any issues?

No

### CODE REVIEW

File: packages/next-app/src/components/ClaimCard.tsx [hunk 1]
```diff
@@ -175,6 +188,57 @@ export const ClaimCard = (props: { data: ClaimCardData }) => {
...
+                message="Claim succesful - "
+                link_message="view on etherscan"
+                link="https://google.com"
+              />
+            )
+          })
+        }
+      >
+        Confirm toast
+      </Button>
```
Comment (janndriessen) [inline packages/next-app/src/components/ClaimCard.tsx:241]: This cool for testing but shouldn't be in the PR in my opinion. You could add a #testing section in the description of the PR.

File: packages/next-app/src/components/ClaimCard.tsx [hunk 2]
```diff
@@ -1,10 +1,22 @@
...
+import { ErrorToast } from "./toasts/error";
+import { InfoToast } from "./toasts/info";
+import { ConfirmToast } from "./toasts/confirm";
 
 export enum ClaimCardState {
   disconnected,
   unclaimed,
   isClaiming,
-  claimed,
+  claimed
```
Comment (janndriessen) [inline packages/next-app/src/components/ClaimCard.tsx:19]: Not sure if we currently have a prettier config in place but this changes shouldn't be checked in, unless there is a specific reason?

Comment (embiem) [reply packages/next-app/src/components/ClaimCard.tsx:19]: Good point, I think we should enforce prettier actually.

With it enabled, I also usually like to keep trailing commas so any new parameter won't show up as changed in the line above.

Comment (embiem) [reply packages/next-app/src/components/ClaimCard.tsx:19]: In general for this PR best keep ClaimCard untouched. We have another PR in the queue that will modify it, which will lead to merge conflicts then.

**Comment by embiem** [comment]
Hello and thanks for the contribution @andreivdev!!

Please see the comments above. Generally please remove the changes to the ClaimCard.

Otherwise, the toast looks good! LGTM after the ClaimCard changes have been removed.

**Comment by andreivdev** [comment]
Thanks for the comments! I removed the changes from ClaimCard.tsx. There still is a new line difference at the end of the file but I couldn't get rid of that. I don't think it will cause conflicts though.

**Pull request merged**